packages/remote-view/README.md

```diff
@@ -0,0 +1,296 @@
+# `<RemoteView />`
```

Comment:
I'd just leave a small intro here and link to the new page in the Modular docs. As we iterate, this will go out of sync quickly. One day, we'll create docs from READMEs and we won't have this problem.